### PR TITLE
invite button: initial GUI work

### PR DESCRIPTION
**Pull request by jakob223**
cc @keybase/y2ksquad 
image
image
image

### CODE REVIEW

**Comment by buoyad** [comment]
`people/invite-friends` seems to be missing here

File: shared/people/index.native.tsx [hunk 1]
```diff
@@ -20,12 +20,17 @@ export const Header = (props: Props) => (
...
+  <>
+    <Kb.ScrollView
+      style={styles.scrollView}
+      refreshControl={
+        <Kb.NativeRefreshControl refreshing={props.waiting} onRefresh={() => props.getData()} />
+      }
+    >
+      <PeoplePageList {...props} />
+    </Kb.ScrollView>
+    <InviteFriends />
```
Comment (buoyad) [inline shared/people/index.native.tsx]: feature flag

File: shared/people/invite-friends/tab-bar-button.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      <Kb.Text type="BodySmallBold" style={styles.counter}>
+        {num.toLocaleString()} {showFire ? <Kb.Emoji emojiName="fire" size={12} /> : null}
+      </Kb.Text>
+    </Kb.Box2>
+  )
+  return Styles.isMobile ? (
+    <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.mobileContainer}>
+      <Kb.Box style={Styles.globalStyles.flexOne} />
+      {inviteButton}
+      {inviteCounter}
```
Comment (buoyad) [inline shared/people/invite-friends/tab-bar-button.tsx]: Designs say to display a tooltip on tap here

File: shared/people/invite-friends/tab-bar-button.tsx [hunk 3]
```diff
@@ -23,11 +23,12 @@ const InviteFriends = () => {
       </Kb.Text>
     </Kb.Box2>
   )
+  const firstTooltipLine = `${num.toLocaleString()} friends invited in the last 24 hours.`
   return Styles.isMobile ? (
     <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.mobileContainer}>
       <Kb.Box style={Styles.globalStyles.flexOne} />
       {inviteButton}
-      {inviteCounter}
+      <Kb.WithTooltip tooltip={firstTooltipLine}>{inviteCounter}</Kb.WithTooltip>
```
Comment (buoyad) [inline shared/people/invite-friends/tab-bar-button.tsx]: Something is off with this layout on mobile now :/ 
image

Comment (jakob223) [reply shared/people/invite-friends/tab-bar-button.tsx]: oops thanks for thorough testing
fixed now, boxes within boxes within boxes

**Pull request merged**